docs/guides/chip_tool_guide.md

```diff
@@ -0,0 +1,807 @@
+In this command:
+
+-   _<node_id\>_ is the user-defined ID of the node being commissioned.
+-   _<ssid\>_ and _<password\>_ are credentials determined in the step 3.
+-   _<pin_code\>_ and _<discriminator\>_ are device-specific keys determined in
+    the step 4.
+
+If you prefer the hexadecimal format, use the `hex:` prefix. For example:
+
+```
+$ ./chip-tool pairing ble-wifi <node_id> hex:<ssid> hex:<password> <pin_code> <discriminator>
+```
+
+> **Note:** The _<node_id>_ can be provided as a hexadecimal value with the `0x`
+> prefix.
+
+#### Commissioning into a network over IP
+
+This option is available when the Matter device is already present in an IP
+network, but it has not been commissioned to a Matter network yet.
+
+##### Commissioning with setup PIN code
+
+Use the following command pattern to discover devices and try to pair with the
+first discovered one using the provided setup code:
+
+```
+$ ./chip-tool pairing onnetwork <node_id> <pin_code>
+```
+
+In this command:
+
+-   _<node_id\>_ is the user-defined ID of the node being commissioned.
+-   _<pin_code\>_ is device specific _setup PIN code_ determined in the step 4
+    and used to discover the device.
+
+##### Commissioning with long discriminator
+
+Use the following command pattern to discover devices with a long discriminator
+and try to pair with the first discovered one using the provided setup code.
+
+```
+$ ./chip-tool pairing onnetwork-long <node_id> <pin_code> <discriminator>
+```
+
+In this command:
+
+-   _<node_id\>_ is the user-defined ID of the node being commissioned.
+-   _<pin_code\>_ and _<discriminator\>_ are device specific keys determined in
+    the step 4.
+
+##### Commissioning with QR code payload
+
+Matter devices log the QR code and its payload when they boot.
+
+Use the following command pattern to discover devices based on the given QR code
+payload and try to pair with the first discovered one:
+
+```
+$ ./chip-tool qrcode <node_id> <qrcode_payload>
+```
+
+In this command:
+
+-   _<node_id\>_ is the user-defined ID of the node being commissioned.
+-   _<qrcode_payload\>_ is the QR code payload ID, for example
+    `MT:Y.K9042C00KA0648G00`.
+
+#### Forgetting the already-commissioned device
+
+The following command removes the device with the given node ID from the list of
+commissioned Matter devices:
+
+```
+$ ./chip-tool pairing unpair <node_id>
+```
+
+In this command, _<node_id\>_ is the user-defined ID of the node which is going
+to be forgotten by the CHIP Tool.
+
+### Step 7: Control application Data Model clusters
+
+Having completed all previous steps, you have the Matter device successfully
+commissioned to the network. You can now test the device by interacting with
+Data Model clusters.
+
+For instance, in case of the lighting app example, the application has `onoff`
+and `levelcontrol` clusters implemented. This means that you can test by
+toggling the bulb (using `onoff`) or manipulating its brightness (using
+`levelcontrol`).
```

Comment:
```suggestion
For instance, in case of the lighting application, the application has the On/Off
and Level Control clusters implemented. This means that you can test it by
toggling the bulb (using the ``onoff`` cluster commands) or manipulating its brightness (using
the ``levelcontrol`` cluster commands).
```